sorting/quick_sort.cpp

```diff
@@ -19,83 +20,219 @@
  *      than x) before x, and put all greater elements (greater than x) after
  *      x. All this should be done in linear time
  *
+ * @author [David Leal](https://github.com/Panquesito7)
+ * @author Unknown author
```

Comment:
Non-blocking; I think you can figure it out if you check out the commit `f22baf19210b17c8567d461ad35259322543701c` in your local - I don't have this repo cloned in my local env so can't do this. This was the parent commit before `ayankhan` made his changes to fix overflow issues, once you check that out you can do `git blame sorting/quick_sort.cpp` to check who wrote it 😄.

Reply:
Nice, thanks! 👍 
I found the author's name within GitHub's UI. I just checked that commit, clicked on `Browse files`, searched for the quick sort file, and found it. However, the folders and the filename were renamed, thus, the history being rewritten/lost; thankfully, GitHub has the option to check older history, which then led me to the original author submitting the algorithm.